wagtailmodelchoosers/client/DefaultDecorator.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+type Props = {
+    entityKey: string;
+    contentState: any;
+    children: React.ReactNode;
+};
```

Comment:
Should be available in draft-js types which you already added to package.json https://github.com/facebook/draft-js/blob/74ebc510be96c5d850bde71f5e9f609636fd304f/src/model/decorators/DraftDecorator.js#L54

Reply:
@liamjohnston

Reply:
oh also i think you missed draftailmodelchoosers.js

Reply:
also i rewrote the history a bit.  do this:
```
git fetch
git checkout upgrade_to_draftail
git reset origin/upgrade_to_draftail --hard
```

Reply:
@liamjohnston if you already started work before doing the above then tell me - we'll sort it out.

Reply:
unless you're feeling pro at git

Reply:
>oh also i think you missed draftailmodelchoosers.js

@aaronhaslett whoops! I accidentally did `wagtailmodelchoosers.js` instead... no harm done I guess? Will do `draftailmodelchoosers.js` now